db/version_set.cc

```diff
@@ -1534,6 +1535,37 @@ void Version::GetCreationTimeOfOldestFile(uint64_t* creation_time) {
   *creation_time = oldest_time;
 }
 
+Status Version::TryVerifySstUniqueIds() {
```

Comment:
In the comment for `ColumnFamilyData::TryVerifySstUniqueIds`, you mentioned that this function requires db mutex. Does it have to hold the db mutex for the entire duration? Looks like right now, unopened files are not verified, is this going to be the case in the future?

Reply:
Good point, we changed to verify all files (open all files), so added unlock before verification.